packaging/suse/amazon-ecs-init.spec

```diff
@@ -12,37 +12,141 @@
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
 
-# Please submit bugfixes or comments via http://bugs.opensuse.org/
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
-# TODO: replace with generic rpm. Need to verify the latter works on EC2.
+
 
 %define short_name amazon-ecs
 Name:           amazon-ecs-init
-Version:        1.7.1
+Version:        1.52.1
 Release:        0
-Summary:        Amazon Elastic Container Service Initialization
+Summary:        Amazon EC2 Container Service Initialization
 License:        Apache-2.0
 Group:          System Environment/Base
-Url:            https://github.com/aws/amazon-ecs-init
-Source0:        %{name}-%{version}.tar.gz
+URL:            https://github.com/aws/amazon-ecs-init
+Source0:        %{name}-%{version}-1.tar.gz
 Source1:        %{short_name}.service
-BuildRequires:  go
-BuildRequires:  systemd
-Requires:       docker >= 1.6.0
+# Patch local to openSUSE Build service to get reproducible builds
+#Patch0:         reproducible.patch
```

Comment:
Is this patch something that can be upstreamed so that ecs-init builds reproducibly?

Reply:
@rjschwei Is this something you can respond to?